### PR TITLE
Feature/prep dao

### PR DESCRIPTION
**Pull request by Velenir**
This PR is intended to be for the actual DAO deployment

Only changes for now are to env variables in `env_vars/dao.env`
Config is `src/config` not changed, only added comments to easily see calculated vars

Dry run log is available in ` deployment-mainnet-dryrun.txt`

### CODE REVIEW

**Comment by orenyodfat** [comment]
does this pr include the parameters for the real dxDAO  ?

File: contracts/utils/TokenRegistry.sol [hunk 1]
```diff
@@ -120,7 +120,7 @@ contract TokenRegistry is Ownable {
         tokenAddresses.push(_token);
```
Comment (orenyodfat) [inline contracts/utils/TokenRegistry.sol]: what is the purpose of the this contract?

File: package.json [hunk 2]
```diff
@@ -43,22 +43,22 @@
...
     "debug": "^4.1.1",
-    "prettier": "^1.16.3",
+    "prettier": "^1.16.4",
     "prettier-plugin-solidity-refactor": "^1.0.0-alpha.14",
     "scrypt": "^6.0.3",
     "truffle-contract": "^4.0.4",
     "truffle-hdwallet-provider": "^1.0.3"
   }
-}
+}
```
Comment (orenyodfat) [inline package.json]: please remove CR

File: contracts/utils/TokenRegistry.sol (hunk 1 above)
Comment (anxolin) [reply contracts/utils/TokenRegistry.sol]: It's used by one script to get the token list of 0x. Not currently used, but is useful to keep

**Comment by anxolin** [comment]
> please mention which issues this pr is fixing

All of the ones u opened :) 

https://github.com/gnosis/dx-daostack/issues/57
https://github.com/gnosis/dx-daostack/issues/55
https://github.com/gnosis/dx-daostack/issues/54
https://github.com/gnosis/dx-daostack/issues/53

Anyways, the purpose of the branch is settle the config params for dxDAO and this test

File: package.json (hunk 2 above)
Comment (anxolin) [reply package.json]: I don't understand the issue

**Pull request merged**